src/invoice-example.xml

```diff
@@ -0,0 +1,20 @@
+<invoice>
+    <invoice-number>7c88ef81e7714b5fbd64434d4c1dd2ef</invoice-number>
+    <date>2023-10-01</date>
+    <company>
+        <name>Drying Paint Inc.</name>
+        <address>Paintburg, Paint Street 54/6</address>
+        <NIP>123-456-78-90</NIP>
+        <bank-account-number>12 3456 7890 0000 0000 1234 5678</bank-account-number>
+    </company>
+    <client>
+        <name>John</name>
+        <surname>Smith</surname>
+        <mail>john@smi.th</mail>
+        <subscriptions>bronze</subscriptions>
+    </client>
+    <payment>
+        <amount>29.99</amount>
+        <due>2023-11-01</due>
+    </payment>
```

Comment:
Should we maybe add the payment method? For example VISA/Mastercard - 1234?

Reply:
I checked one of my own invoices and there isn't anything about the payment method, except for the bank account number and payment title (which is the same as the invoice number)

Reply:
ok, maybe it doesn't have to be there. While generating text for sendinv invoices i was inspired by this image: https://verifone.cloud/sites/default/files/verifone_2checkout/relative/Documentation/11Emails/02Default_shopper_emails/Invoice_email/invoice%252Bemail.png